stdlib/public/Concurrency/Task.swift

```diff
@@ -312,6 +330,22 @@ extension Task {
       }
     }
 
+    /// Whether this is a task created by the 'async' operation, which
+    /// conceptually continues the work of the synchronous code that invokes
+    /// it.
```

Comment:
oh okey, hm... i don't love the "continuing name" but it's better than no qualifier...

Will use this wording in the proposal too then 👍